core/src/main/java/nu/yona/server/subscriptions/service/BuddyDTO.java

```diff
@@ -88,7 +88,7 @@ private String determineTempNickname(Translator translator)
 
 	public static BuddyDTO createInstance(Buddy buddyEntity)
 	{
-		return new BuddyDTO(buddyEntity.getID(), UserDTO.createInstanceIfNotNull(buddyEntity.getUser()),
+		return new BuddyDTO(buddyEntity.getID(), UserDTO.createInstance(buddyEntity.getUser()),
```

Comment:
Why did you change this line? It may be that the buddy user is removed, isn't it?